app/stac_api/validators.py

```diff
@@ -187,3 +187,29 @@ def validate_item_properties_datetimes(
             properties_start_datetime,
             properties_end_datetime,
         )
+
+
+def validate_JSON_payload_write_requests(serializer_class, request_data):  # pylint: disable=invalid-name
```

Comment:
please stick to the naming convention, all lower case and don't disable invalid-name

Reply:
I knew it! ;-)